fix_lint.sh

```
@@ -0,0 +1,4 @@
+#!/bin/bash
+# Apply some automated fixes
+cd frontend
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)